shaders-msl/asm/frag/op-image-sampled-image.asm.frag

```diff
@@ -0,0 +1,495 @@
+; SPIR-V
```

Comment:
This is a pretty huge test shader for something that could be simple.
Is it possible to make a simpler test shader in asm?